src/main/java/io/fabric8/jenkins/openshiftsync/OpenShiftUtils.java

```diff
@@ -84,6 +91,18 @@ public synchronized static OpenShiftClient getOpenShiftClient() {
     return openShiftClient;
   }
 
+  // Get the current OpenShiftClient and configure to use the current Oauth token.
+  public synchronized static OpenShiftClient getAuthenticatedOpenShiftClient() {
+    if(openShiftClient != null){
+      String token = CredentialsUtils.getCurrentToken();
+      if(token.length() > 0) {
+        openShiftClient.getConfiguration().setOauthToken(token);
```

Comment:
Because getCurrentToken has to grab the credentialsId from the GlobalConfig, we still need to have an additional getOpenShiftClient method for when the GlobalConfig is first loaded